Packs/Netskope/Integrations/NetskopeEventCollector/NetskopeEventCollector.yml

```diff
@@ -17,28 +17,19 @@ configuration:
   required: true
   type: 9
   section: Connect
-- additionalinfo: The API version to use (v1 or v2).
-  defaultvalue: v2
-  display: API Version
-  name: api_version
-  type: 15
-  section: Connect
-  options:
-  - v1
-  - v2
-  required: false
 - additionalinfo: 'The first fetch is limited to the last 50,000 events (for each event type).'
-  defaultvalue: 3 days
+  defaultvalue: now
   display: First fetch timestamp (<number> <time unit>, e.g., 12 hours, 7 days)
   name: first_fetch
-  section: Collect
   type: 0
+  section: Collect
   required: false
-- additionalinfo: The maximum amount of events to retrieve for each event type (up to 30000 events). For more information about event types see the help section.
-  defaultvalue: '1000'
+  hidden: true
+- additionalinfo: 'The maximum amount of events to retrieve PER EACH EVENT TYPE type. For more information about event types see the help section.'
+  defaultvalue: '50000'
```

Comment:
Don't you prefer to lower the default value?

Reply:
Why actually?

Reply:
To not take risks, not to overwhelm the system too much if it is not necessary.
No reason not to start with a bit lower value and if the client wants, he can increase (30K for example) but whatever you think

Reply:
Don't see a reason we can digest 50k with no problem. We also have the timeout mechanism for extra safety